src/IECoreHoudini/GEO_CortexPrimitive.cpp

```diff
@@ -455,18 +500,25 @@ void GEO_CortexPrimitive::copyPrimitive( const GEO_Primitive *src )
 	}
 	
 	const GEO_CortexPrimitive *orig = (const GEO_CortexPrimitive *)src;
-	const GA_IndexMap &srcPoints = orig->getParent()->getPointMap();
-	
+
 	/// \todo: should we make a shallow or a deep copy?
 	m_object = orig->m_object;
+
+	GEO_Primitive::copyPrimitive(src);
```

Comment:
I'll put this back in case it's necessary for H15, but not call it for H16. It manages the vertex list in H16.